code/modules/mob/mob.dm

```diff
@@ -1172,7 +1172,7 @@ var/list/slot_equipment_priority = list( \
 		memory()
 
 //mob verbs are faster than object verbs. See http://www.byond.com/forum/?post=1326139&page=2#comment8198716 for why this isn't atom/verb/examine()
-/mob/verb/examination(atom/A as mob|obj|turf in view()) //It used to be oview(12), but I can't really say why
+/mob/verb/examination(atom/A as mob|obj|turf in view(loc)) //It used to be oview(12), but I can't really say why
```

Comment:
According to the Ref, `view(client)` works, and means anything the player can see. Can you test that? Because if it works as described, it sounds perfect.

Reply:
What circumstance would you be in 2 nested objects in?

Reply:
Tested. Right click examine doesn't work when in a locker.

Reply:
SGuivenne an example given was in a bodybag in a morgue, or as a pAI (The mob is in the card, which is an object) in a PDA

Reply:
@SGuivenne Humans, rarely if ever unless something goes wrong. Any creature that can be picked up, regularly.
@MadmanMartian Wonderful. Glad the Ref is helpful as ever. What about `view(get_turf(client.eye), client.view)`?

Reply:
It works, but it lets me examine things as an AI.

Reply:
So you're telling me the only thing stopping AIs from examining was a bug? Fucking excellent
Whatever, using `client.eye` was a bad idea anyway. Just replace that with `src` and I'll stop pestering you.